Makefile

```diff
@@ -200,6 +200,7 @@ data-flow-diagram: docs/data-flow.dot
 clean:
 	go clean -i ./...
 	rm -rf *.out
+	rm -rf bin
```

Comment:
Is it better just clean files that are generated by ticdc

Reply:
I never run `make clean`, hahaha

Reply:
In general, the cleanup command should also clean up the generated binaries?

Reply:
For example: https://github.com/kubernetes/kops/blob/2f003d8e5664d403889c831a69578c74ebb93507/Makefile#L140

https://github.com/tikv/tikv/blob/45ce5b9584d618bc777877be8a77cb94f61b8410/Makefile#L146

So I think it might be reasonable to clean up the generated binaries here. 😄

Reply:
Yep what I mean is there could be third party binaries in `bin` directory, which could be used in integration test, ref: https://github.com/pingcap/ticdc/blob/d933dd17544711f6fed721723270bb4408882d7e/Makefile#L119-L129

We can make it more standardized about which directory will output build targets and could be cleaned by `make clean`

Reply:
Oh, I missed those targets, sorry! I will clean up only the files generated by the cdc compilation.